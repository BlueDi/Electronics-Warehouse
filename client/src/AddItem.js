import React, { Component } from "react";
import "./App.css";
import { Link } from "react-router-dom";
import { Grid, Button, Icon, GridColumn } from "semantic-ui-react";
import TableExample from "./Table";
import WHSearchBar from "./SearchBar"

class AddItem extends Component {

    render() {
        return (

            <div style={{ paddingTop: "1em" }}>
                <div style={{ float: "right" }}>
                    <WHSearchBar key="searchBar" />
                </div>

                <div>
                    <Link to="/addItem">

                    </Link>
                </div>

                <div style={{ paddingTop: "2em" }}>
                    <TableExample />
                </div>
            </div>
        );
    }
}

export default AddItem;
