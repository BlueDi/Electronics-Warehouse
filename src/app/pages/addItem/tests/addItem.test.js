import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import AddItem from '@pages/addItem';

test('<AddItem />', () => {
  const component = renderer.create(
    <Router>
      <AddItem />
    </Router>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
