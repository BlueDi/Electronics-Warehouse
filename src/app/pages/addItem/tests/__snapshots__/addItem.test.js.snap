// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddItem /> 1`] = `null`;
